src/server/game/AI/SmartScripts/SmartScriptMgr.h

```diff
@@ -694,8 +694,9 @@ enum SMART_ACTION
     SMART_ACTION_ATTACK_STOP                        = 224,    //
     SMART_ACTION_SET_GUID                           = 225,    // Sends the invoker's or the base object's own ObjectGuid to target
     SMART_ACTION_DISABLE                            = 226,    // Disable the targeted creatures, setting them Invisible and Immune to All
+    SMART_ACTION_SET_SCALE                          = 227,    // Modifies the target's scale
```

Comment:
personally I think the comment should be reserved to what the fields are. Like
`SMART_ACTION_RANDOM_MUSIC                       = 217,    // SoundId1, SoundId2, SoundId3, SoundId4, onlySelf, type`